Replicator/tests/SG.cc

```diff
@@ -123,11 +124,10 @@ alloc_slice SG::addChannelToJSON(slice json, slice ckey, const std::vector<std::
 bool SG::createUser(const std::string& username, const std::string& password,
                     const std::vector<std::string> &channelIDs) const {
```

Comment:
Where is "channelIds" used in this function?

Reply:
Ah you’re right, I’ve removed the use from this function now